build_tools/azure/render_meta.py

```diff
@@ -0,0 +1,52 @@
+import os
+import sys
+
+from jinja2 import Environment, FileSystemLoader
+
+# Constant file paths. Relative to this file, so we have to run this file from this directory
+DIST_PATH = '../../dist'
+VERSION_FILE = '../../VERSION'
+REQUIREMENTS_FILE = '../../requirements.txt'
+OUTPUT_DIR = '../../conda'
+OUTPUT_FILE = '../../conda/meta.yaml'  # conda is weird about yml vs yaml, so we have to use yaml
```

Comment:
I get a little bit of heartburn seeing rel paths only. Someone will invariably mess up someday. Could we add:

```python
ABSDIR = os.path.abspath(os.path.dirname(__file__))
```

And make them all, e.g.,

```python
DIST_PATH = ABSDIR + "../../dist"
...
```

Reply:
Yeah, I can definitely do that. `pathlib` was giving me trouble on 3.5 on GitHub Actions, so this is an artifact of that. I can even try to update it to use `pathlib` if we want, since this is now on ADO

Reply:
I changed this to use `pathlib`. Fixed with 006a582415861cbfe9bac8104990770b316bfdce